pkg/monitoring/constants.go

```diff
@@ -3,8 +3,14 @@
 
```

Comment:
Update copyright for 2021

Reply:
Addressed in commit e0cb7e9523fa6287222cc134a03a69a0cfd26fa2